### DISCUSSION TITLE
Renovate create a PR with invalid pom.xml

### DISCUSSION THREAD

**Discussion by modiboa** [renovatebot/renovate#18521] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.241.11

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.4.2-ee

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Renovate creates a PR that broken the `pom.xml`. The wierd part is that the Renovate already updates the same package, but this time, the PR was wrong.

Can anyone help me to understand what is happening here? Renovate seems to identify the correct dependency ( `io.repaint.maven:tiles-maven-plugin` ), but fail to edit the file.

DIFF
![Screenshot from 2022-10-24 16-29-41](https://user-images.githubusercontent.com/6961049/197565356-d1aaf5d7-2593-428f-882b-90032ca436a7.png)

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
pretty sure they're missing git settings on windows. we've documented that in our getting started section.

**Reply by viceice** [reply]
https://docs.renovatebot.com/getting-started/installing-onboarding/#self-hosting-on-windows

**Comment by thiagofabre** [comment]
Hey guys!

We are using the Renovate docker image with gitlab pipeline. What is weird is that the same dependency was updated in a previous PR sucessfully, but this time the PR was wrong. 

I'll try to create a small repo to reproduce that

**Comment by PhilipAbed** [comment]
@thiagofabre does this cover your case? 
https://github.com/renovatebot/renovate/issues/18630

**Reply by thiagofabre** [reply]
In my case the Renovate PR only removes the `>` characater. In this case, it removes all property name.
But it seems the both are related
